plugins/statuspage/schema.py

```diff
@@ -44,3 +46,22 @@
     "additionalProperties": False,
     "required": ["component"],
 }
+
+
+# https://developer.statuspage.io/#operation/postPagesPageIdIncidents
+incident_request = {
+    "type": "object",
+    "properties": {
+        "name": {"type": "string", "description": "Incident Name"},
+        "status": {"type": "string", "description": "Incident Status"},
+        "body": {
+            "type": "string",
+            "description": "The initial message, created as the first incident update.",
+        },
+        "components": {
+            "type": "object",
+            "description": "Map of status changes to apply to affected components",
+            "properties": {"component_id": status_type},
```

Comment:
The description and the documented type from the link in the comment don't seem to match, so this is based on their example payloads.